terraform/environments/digital-prison-reporting/athena_federated_queries.tf

```diff
@@ -1,7 +1,7 @@
 locals {
   nomis_host              = jsondecode(data.aws_secretsmanager_secret_version.nomis.secret_string)["endpoint"]
-  nomis_sid               = try(jsondecode(data.aws_secretsmanager_secret_version.nomis_athena_federated.secret_string)["sid"], "")
-  connection_string_nomis = "oracle://jdbc:oracle:thin:$${${aws_secretsmanager_secret.nomis_athena_federated.name}}@${local.nomis_host}:1521:${local.nomis_sid}"
+  nomis_service_name      = jsondecode(data.aws_secretsmanager_secret_version.nomis.secret_string)["db_name"]
+  connection_string_nomis = "oracle://jdbc:oracle:thin:$${${aws_secretsmanager_secret.nomis.name}}@//${local.nomis_host}:1521/${local.nomis_service_name}"
```

Comment:
This is the main change. It's required since we use service names in environments above test rather than sids